packages/firebase_crashlytics/firebase_crashlytics/ios/firebase_crashlytics.podspec

```diff
@@ -34,7 +34,7 @@ Pod::Spec.new do |s|
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
 
-  s.ios.deployment_target = '9.0'
+  s.ios.deployment_target = '10.0'
```

Comment:
Looks like Crashlytics is still 9.0 -> https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCrashlytics.podspec#L14 - but Analytics is definitely 10 at least 😅

Reply:
Woops, I've reverted 👌